src/test/kotlin/com/atlassian/performance/tools/hardware/report/RecommendationsTable.kt

```diff
@@ -0,0 +1,53 @@
+package com.atlassian.performance.tools.hardware.report
+
+import com.atlassian.performance.tools.hardware.HardwareTestResult
+import com.atlassian.performance.tools.hardware.RecommendationSet
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVPrinter
+import java.io.File
+import java.nio.file.Path
+import java.time.Duration
+
+class RecommendationsTable {
+
+    fun tabulate(
+        recommendations: RecommendationSet,
+        output: Path
+    ): File {
+        val headers = arrayOf(
+            "reason",
+            "jira",
+            "jira nodes",
+            "database",
+            "apdex average (0.0-1.0)",
+            "cost [USD/hour]"
+        )
+        val format = CSVFormat.DEFAULT
+            .withHeader(*headers)
+            .withRecordSeparator('\n')
+        val tableFile = output.toFile()
+        tableFile.bufferedWriter().use { writer ->
+            val printer = CSVPrinter(writer, format)
+            printRecommendation(recommendations.bestApdex, "best Apdex and reliabilty", printer)
```

Comment:
It would be nicer to have other results also included in the table, so we can see why/how those recommended ones are chosen.

Reply:
The recommendation charts fulfill that purpose.
This is supposed to be easy for the IX to consume.